### PR TITLE
Newsletters: add 264 (2023-08-16)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, and topic entries @harding

### CODE REVIEW

File: _posts/en/newsletters/2023-08-16-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    would support expiry dates up to about 45,000 years from now.
+
+    The recommendation received a moderate amount of discussion on the
+    mailing list, with no clear resolution as of this writing.
+
+- **Serverless payjoin:** Dan Gould [posted][gould spj] to the
+  Bitcoin-Dev mailing list a [draft BIP][spj bip] for _serverless
+  [payjoin][topic payjoin]_.  By itself, [payjoin][topic payjoin] as
+  specified in [BIP78][] expects the receiver to operate a server for
+  securely accepting PSBTs from spenders.  Gould proposes an asynchronous
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-08-16-newsletter.md:41]:
```suggestion
  securely accepting [PSBTs][topic psbt] from spenders.  Gould proposes an asynchronous
```

File: _posts/en/newsletters/2023-08-16-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    inclusion of either a two-byte expiry time in addresses that would
+    support expiry dates up 180 years from now or a three-byte time that
+    would support expiry dates up to about 45,000 years from now.
+
+    The recommendation received a moderate amount of discussion on the
+    mailing list, with no clear resolution as of this writing.
+
+- **Serverless payjoin:** Dan Gould [posted][gould spj] to the
+  Bitcoin-Dev mailing list a [draft BIP][spj bip] for _serverless
+  [payjoin][topic payjoin]_.  By itself, [payjoin][topic payjoin] as
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-08-16-newsletter.md:39]: Maybe can drop the second topic link

Comment (bitschmidty) [inline _posts/en/newsletters/2023-08-16-newsletter.md:39]: Consider a (see Newsletter #236) link to https://bitcoinops.org/en/newsletters/2023/02/01/#serverless-payjoin-proposal

**Comment by harding** [comment]
Made edits for @bitschmidty feedback (thanks!), added lede, releases/RCs, and topic entries.  I mentioned Brandon's article in the lede but left a FIXME for @bitschmidty to remove that mention if it doesn't get added in this week's newsletter.

Thanks everyone!

File: _topics/en/eclipse-attacks.md [hunk 3]
```diff
@@ -77,6 +77,9 @@ optech_mentions:
   - title: LND 0.14.0-beta includes additional eclipse attack protection by sharing block headers
     url: /en/newsletters/2021/11/24/#lnd-0-14-0-beta
 
+  - title: "Bitcoin Core #27213 will attempt to open and maintain one connection on each reachable network"
```
Comment (bitschmidty) [inline _topics/en/eclipse-attacks.md:80]: @harding would it also be a valuable topic entry for https://bitcoinops.org/en/topics/anonymity-networks/ ?

File: _includes/articles/bitgo-musig2.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+There are two main
+benefits of MuSig compared to script multisig. The first and most obvious
+benefit is a reduced transaction size and miner fee. Onchain signatures are
+64-73 bytes, 16-18.25 virtual bytes (vB), and MuSig can combine two (or more)
+signatures into one. In BitGo’s 2-of-3 case, a MuSig key path input costs
+57.5vB, compared to a native SegWit input at 104.5vB or depth 1
+[tapscript][topic tapscript] at
+107.5vB. The second benefit of MuSig is an improvement in privacy. With a MuSig
+key path on a collaboratively held output, a cooperative spend cannot be
+distinguished by a third party blockchain observer from a single signature
```
Comment (LarryRuane) [inline _includes/articles/bitgo-musig2.md:29]:
```suggestion
distinguished by a third-party blockchain observer from a single signature
```

File: _includes/articles/bitgo-musig2.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+by allowing one of their signers not to manage state. By reviewing (and
+implementing) the draft specification during its development, we were able to
+contribute to the specification and be ready to launch MuSig2 signing soon after
+the specification was formally published as BIP327.
+
+### MuSig and PSBTs
+
+The [PSBT (Partially Signed Bitcoin Transaction)][topic psbt] format is intended to carry all
+information needed to sign a transaction between the parties (e.g., coordinator
+and signers in a simple case). The more information is needed for signing, the
```
Comment (LarryRuane) [inline _includes/articles/bitgo-musig2.md:145]:
```suggestion
and signers in a simple case). The more information needed for signing, the
```
Or maybe "The more information that is needed for signing" ... I can't decide. I think I prefer the first suggestion.

File: _includes/articles/bitgo-musig2.md [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+transaction data interchange, and it’s been a huge success. It’s not widely
+known yet, but BitGo wallets (except those using MuSig2, see the next paragraph)
+can now integrate with hardware signing devices that support PSBTs.
+
+There are not yet published PSBT fields for use in MuSig2 signing. For our
+implementation, we used proprietary fields that were based on a draft shared
+with us by [Sanket][]. This is one of the little talked about benefits of the PSBT
+format - the ability to include _whatever_ additional data might be needed for
+your custom transaction building or signing protocol in the same binary data
+format; with global, per-input, and per-output sections already defined. The
```
Comment (LarryRuane) [inline _includes/articles/bitgo-musig2.md:158]:
```suggestion
format, with global, per-input, and per-output sections already defined. The
```
(Usually the part after a semicolon could stand alone as a sentence.)

File: _includes/articles/bitgo-musig2.md [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+more valuable the format becomes. We examined the costs and benefits of
+expanding our existing API format with additional fields to facilitate MuSig2
+signing vs. converting to PSBT. We settled on converting to PSBT format for
+transaction data interchange, and it’s been a huge success. It’s not widely
+known yet, but BitGo wallets (except those using MuSig2, see the next paragraph)
+can now integrate with hardware signing devices that support PSBTs.
+
+There are not yet published PSBT fields for use in MuSig2 signing. For our
+implementation, we used proprietary fields that were based on a draft shared
+with us by [Sanket][]. This is one of the little talked about benefits of the PSBT
```
Comment (LarryRuane) [inline _includes/articles/bitgo-musig2.md:155]:
```suggestion
with us by [Sanket][]. This is one of the little talked-about benefits of the PSBT
```

File: _posts/en/newsletters/2023-08-16-newsletter.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+permalink: /en/newsletters/2023/08/16/
+name: 2023-08-16-newsletter
+slug: 2023-08-16-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about adding expiration
+dates to silent payment addresses and provides an overview of a draft BIP
+for serverless payjoin.  A contributed field report describes
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-16-newsletter.md:12]:
```suggestion
for serverless payjoin.  A contributed field report describes the
```

File: _posts/en/newsletters/2023-08-16-newsletter.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+
+- **Serverless payjoin:** Dan Gould [posted][gould spj] to the
+  Bitcoin-Dev mailing list a [draft BIP][spj bip] for _serverless
+  payjoin_ (see [Newsletter #236][news236 spj]).  By itself, [payjoin][topic payjoin] as
+  specified in [BIP78][] expects the receiver to operate a server for
+  securely accepting [PSBTs][topic psbt] from spenders.  Gould proposes an asynchronous
+  relay model that would start with a receiver using a [BIP21][] URI to
+  declare the relay server and symmetric encryption key they wanted to
+  use for receiving payjoin payments.  The spender would encrypt a PSBT
+  of their transaction and submit it the receiver's desired relay.  The
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-16-newsletter.md:51]:
```suggestion
  of their transaction and submit it to the receiver's desired relay.  The
```

File: _posts/en/newsletters/2023-08-16-newsletter.md [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+  receiver would download the PSBT, decrypt it, add a signed input to
+  it, encrypt it, and submit it back to the relay.  The spender
+  downloads the revised PSBT, decrypts it, ensures that it is correct,
+  signs it, and broadcasts it to the Bitcoin network.
+
+    In a [reply][gibson spj], Adam Gibson warned about the danger of
+    including the encryption key in the BIP21 URI and about the risk to
```
Comment (LarryRuane) [inline _posts/en/newsletters/2023-08-16-newsletter.md:58]:
```suggestion
    including the encryption key in the BIP21 URI and the risk to
```

File: _topics/en/eclipse-attacks.md (hunk 3 above)
Comment (harding) [reply _topics/en/eclipse-attacks.md:80]: I thought about that but decided against it.  This is really about eclipse protection and preventing partitions, which takes advantage of those networks when they're available but isn't special to those networks (e.g. it also works on ipv4 vs ipv6).

**Pull request merged**